package.json

```
@@ -242,7 +242,8 @@
     "@types/react-dom": "18.2.4"
   },
   "engines": {
-    "node": ">=16.8.0"
+    "node": ">=16.8.0",
+    "pnpm": "8"
```

```suggestion
    "pnpm": "8.6.11"
```

Should we be more specific here in case someone doesn't have corepack enabled?